### ISSUE TITLE
All implementations of abstract methods are considered unused

### ISSUE THREAD

**Issue by terifan** [apache/netbeans#4276]
### Apache NetBeans version

Apache NetBeans 14

### What happened

Obvious bug that make code harder to read. Was working in previous version.

### How to reproduce
```java
package test;

public class Test
{
	public static void main(String[] args)
	{
		Abstr impl = new Impl();
		impl.get();
	}
}

abstract class Abstr
{
	abstract void get();
}

class Impl extends Abstr
{
	@Override
	void get() // <---- considered unused
	{
		System.out.println("hello world");
	}
}
```

### Did this work correctly in an earlier version?

Apache NetBeans 13

### Operating System

Windows 8.1

### JDK

17.0.2

### Apache NetBeans packaging

Apache NetBeans provided installer

### Are you willing to submit a pull request?

No

### Code of Conduct

Yes

**Comment by mbien** [comment]
note: even though this looks like a regression, it isn't since NB didn't have the hint before NB 14 (#3841).

**Comment by neilcsmith-net** [comment]
As mentioned in #4783 this affects non-abstract package-private methods too.  I think we should hide or remove the hint if not fixable by NB16 release @dbalek

**Comment by neilcsmith-net** [comment]
Have noticed that even with the hint disabled, methods are still underlined with an unused message in the tooltip.

**Comment by neilcsmith-net** [comment]
Opened a potential fix for this - cc @mbien @dbalek

**Comment by neilcsmith-net** [comment]
Should be fixed in 16-rc2, available from #4815 Please verify /cc @dbalek
